tools/genesis/generated/genesis_camino.json

```
@@ -0,0 +1,7678 @@
+{
+  "networkID": 1000,
+  "allocations": null,
+  "startTime": 1166713120,
```

```suggestion
  "startTime": 1667131200,
```
Looks like startTime is wrong.